src/js/pages/task-details/TaskDetail.js

```diff
@@ -249,6 +264,31 @@ class TaskDetail extends mixin(InternalStorageMixin, TabsMixin, StoreMixin) {
     );
   }
 
+  getBreadcrumbs() {
+    let innerPath = TaskDirectoryStore.get('innerPath').split('/');
+    let onClickPath = '';
+    let crumbs = innerPath.map((directoryItem, index) => {
+      let textValue = directoryItem;
+
+      // First breadcrumb is always 'Working Directory'.
+      if (index === 0) {
+        textValue = 'Working Directory';
+      } else {
+        onClickPath += ('/' + directoryItem);
```

Comment:
I don't think the breadcrumbs are working as you intended
![img](https://cl.ly/090g0s2r2T0x/Screen%20Recording%202016-10-03%20at%2007.34%20PM.gif)